src/__tests__/TrackVisibility.js

```diff
@@ -32,49 +56,48 @@ describe('<TrackVisibility />', () => {
     });
 
     it('Should fallback clientHeight when window propeties are not defined', () => {
-      const wrapper = mount(<TrackVisibility />);
+      const wrapper = renderComponent(<TrackVisibility />);
       window.innerHeight = null;
       window.innerWidth = null;
       expect(wrapper).toMatchSnapshot();
     });
 
     it('Should bind event to window on mount', () => {
       window.addEventListener = jest.fn();    
-      mount(<TrackVisibility />);
+      renderComponent(<TrackVisibility />);
       expect(window.addEventListener).toHaveBeenCalled();
     });
 
     it('Should remove event from window on unmount', () => {
       window.removeEventListener = jest.fn();    
-      const wrapper = mount(<TrackVisibility />);
+      const wrapper = renderComponent(<TrackVisibility />);
       wrapper.unmount();
       expect(window.removeEventListener).toHaveBeenCalled();
     });
 
     it('Should remove event from window when visibility is tracked once', () => {
       window.removeEventListener = jest.fn();    
-      const wrapper = mount(<TrackVisibility once={true} />);
-      wrapper.setState({isVisible: true});
```

Comment:
I removed this line as it was a little confusing. From what I understand the `componentDidMount` triggers the `isComponentVisible` function which decides the component is visible as `top`, `right`, `bottom`, `left` are all 0